### ISSUE TITLE
Preguntas sobre img2img

### ISSUE THREAD

**Issue by Abel1011** [vpzomtrrfrt/stability-client#1]
Hi, thanks for the library. It works very well so far. I am trying to add the img2img option in my application and reviewing the code I see that you include these parameters:

imagePrompt: {
        mime: string;
        content: Buffer;
    } | null;
    stepSchedule: {
        start?: number;
        end?: number;
    };

Can these values be used to process img2img? What does the stepSchedule represent or what does it do? In the web version of dreamstudio there is a parameter "streng", is it possible to include it in the library?

**Comment by vpzomtrrfrt** [comment]
Yes, these are used for img2img. The API doesn't have any concept of "strength", but it seems to be related to the step schedule. I don't actually know what the values mean
